### DISCUSSION TITLE
Yo any mallu devs here?

### DISCUSSION THREAD

**Discussion by Apollo-Blaze** [community/community#145832] [Programming Help]
Hey there fellow **mallu** devs, you guys know about **bhai lang**??? Well I have made **ettayi lang** (malyalam version). Do check it out and support me at **[https://github.com/Apollo-Blaze/ettayi](url)**

#Python #ProgrammingLanguage

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Apollo-Blaze!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
